home-manager/ime.nix

```diff
@@ -0,0 +1,17 @@
+{ ... }:
+{
+  # https://github.com/nix-community/home-manager/blob/release-24.05/modules/misc/gtk.nix
+  gtk = {
```

Comment:
Removed in https://github.com/kachick/dotfiles/commit/3605181e40b4c680450fd94244738bd1bdcde3c3